README.md

```diff
@@ -45,164 +58,137 @@ $ datapane hello-world
-            dp.DataTable(source)
-    ])
-).save(path="Complex_layout.html", open=True)
+    dp.Formula("x^2 + y^2 = z^2"),
+    dp.Group(
+        dp.BigNumber(
+            heading="Number of percentage points", 
+            value="84%", change="2%", is_upward_change=True
+        ),
+        dp.BigNumber(
+            heading="Simple Statistic", value=100
+        ), columns=2
+    ),
+    dp.Select(
+        dp.Plot(fig, label="Chart"),
+        dp.DataTable(df, label="Data")
+    ),
+).save(path="Layout_example.html")
 ```
+  
+</p>
+</p>
 
-Layout blocks like `dp.Select`, `dp.Group` and `dp.Page` allow you to highlight key points without sacrificing detail, while content blocks like `dp.HTML` and `dp.Formula` (LaTeX) can enrich your report. The final result looks like this:
+<br>
+<br>
+<br>
 
-<img width="1000" alt="Complex Datapane report example" src="https://user-images.githubusercontent.com/16949044/134022445-5d417993-808f-4de8-8e8c-f510bdf4a17e.png">
+# Get involved
 
-Check out the full list of blocks in our [documentation](https://docs.datapane.com/reports/blocks).
+## Discord
 
-# Sharing Reports
+Our Discord community is for people who believe that insights, visualizations, and reports are better created with Python instead of drag-and-drop tools. Get help from the team, share what you're building, and get to know others in the space!
 
-## Sign up for a free account
+### 💬 [Join our discord server](https://chat.datapane.com)
 
-In addition to saving documents locally, you can host, share and embed reports via [Datapane Studio](https://datapane.com/).
+## Forums
+Got feature requests, feedback, or questions? Visit our Forum.
 
-To get your free API key, run the following command in your terminal to sign up via email/OAuth:
+### 📮 [Join our forum](https://forum.datapane.com/)
 
-```
-$ datapane signup
-```
 
-If you're using Jupyter, run `!datapane signup` instead.
+# Hosting Reports
+
+In addition to saving documents locally or hosting them yourself, you can host and share your reports using [Datapane Cloud](https://datapane.com/cloud).
 
-Next, in your Python notebook or script simply change the `save` function to `upload` on your report:
+To get your API key, [create a free account](https://datapane.com/accounts/signup/).
 
+Next, in your Python notebook or script, change the `save` function to `upload`:
+  
 ```python
 dp.Report(
  ...
 #).save(path="hello_world.html")
 ).upload(name="Hello world")
 ```
 
-Your Studio account comes with the following:
-
-- **Unlimited public reports** - great for embedding into places like Medium, Reddit, or your own website (see [here](https://docs.datapane.com/reports/embedding-reports-in-social-platforms))
-- **5 private reports** - share these via email within your organization
-
-### Featured Examples
+# Demos and Examples
 
 Here a few samples of the top reports created by the Datapane community. To see more, check out our [gallery](https://datapane.com/gallery) section.
```

Comment:
the gallery link https://datapane.com/gallery redirects me to https://datapane.com